Revisit page meta descriptions in Core docs - Closes #738 (#753)
LiskHQ/lisk-docs: modules/ROOT/pages/management/source.adoc

= Source code commands
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Source code commands cover all relevant commands required to manage Lisk Core with pm2.

Reviewer: ```suggestion
:description: Covers all relevant commands required to manage Lisk Core from Source with pm2.
```
